ngraph/test/frontend/shared/src/cut_specific_model.cpp

```diff
@@ -216,3 +217,23 @@ TEST_P(FrontEndCutModelTest, testExtractSubgraph) {
                                  }) != ops.end()) << "Name not found:" << name;
     }
 }
+
+TEST_P(FrontEndCutModelTest, testSetTensorValue) {
+    ASSERT_NO_THROW(doLoadFromFile());
+    Place::Ptr place;
+    ASSERT_NO_THROW(place = m_inputModel->getPlaceByTensorName(m_param.m_tensorValueName));
+    ASSERT_NO_THROW(m_inputModel->setTensorValue(place, &m_param.m_tensorValue[0]));
+
+    std::shared_ptr<ngraph::Function> function;
+    ASSERT_NO_THROW(function = m_frontEnd->convert(m_inputModel));
+    auto ops = function->get_ordered_ops();
+
+    auto const_name = m_param.m_tensorValueName;
+    auto const_node_it = std::find_if(ops.begin(), ops.end(),
+                                   [&](const std::shared_ptr<ngraph::Node>& node) {
+                                       return node->get_friendly_name().find(const_name) != std::string::npos;
+                                   });
+    ASSERT_TRUE(const_node_it != ops.end()) << "Name shall exist:" << const_name;
+    auto data = std::dynamic_pointer_cast<opset7::Constant>(*const_node_it)->get_vector<float>();
+    EXPECT_TRUE(std::equal(data.begin(), data.end(), m_param.m_tensorValue.begin())) << "Data must be equal";
```

Comment:
Also would be good to add 'reverse' check, like
EXPECT_TRUE(std::equal(m_param.m_tensorValue.begin(), m_param.m_tensorValue.end(), data.begin())) << "Data must be equal";
Otherwise - when 'data' is empty test will be passed.
Alternatively, you can add check of EXPECT_EQ(data.size(), m_param.m_tensorValue.size())